anagrafica/templates/anagrafica_utente_contatti.html

```diff
@@ -174,7 +174,56 @@ <h4 class="panel-title">
 
                 </div>
 
+                <a name="gsuite"></a>
 
+                <div class="panel panel-info">
+
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <i class="fa fa-fw fa-envelope"></i>
+                            E-mail di servizio
+                            <span class="text-muted">(opzionale)</span>
+                        </h4>
+
+                    </div>
+                    {% if me.volontario %}
+                        {% if me.ha_email_servizio %}
+                            <div class="panel-body">
+                             <p>
+                                Questa è la tua e-mail dell'organizzazione. Potrai decidere dopo averla creata
+                                 se usarla come e-mail di contatto o come e-mail di login.
+                            </p>
+
+                            <pre>{{ me.email_servizio }}</pre>
+
+                            {% if errore_reset_email_servizio %}
+                                <span class="alert alert-warning">{{ errore_reset_email_servizio }}</span>
+                                <br />
+                            {% endif %}
+                            <a href="{% url 'anagrafica-utente_contatti' %}?reset_gsuite_password=1"
+                                class="btn btn-block btn-danger"
+                                onclick="return confirm('Vuoi procedere con il rest della password dell\'email di servizio?')">
```

Comment:
Validazione client side e' da evitare (anche se solo per chiedere conferma). Si puo' sostituire con un modulo separato, oppure con un altro bottone (rosso?) sullo stesso modulo. Per sicurezza, questo tipo di azione e' fondamentale che sia effettuata tramite richiesta POST (quindi con validazione CSRF). Credo che il sistema per il reset della password di Gaia sia buona ispirazione

Reply:
ho cambiato un po' questa parte. Ho messo il check chiedendo di riscrivere l'indirizzo (stile delete repo) preservando però il confirm per un di più.

Reply:
@acarmisc Ok, ma questo tipo di azione deve essere effettuata tramite azione POST con validazione CSRF. La motivazione e' la stessa di sopra: questa soluzione non ci protegge da attacchi di click jacking, nonche' attacchi CSRF. Continua ad essere validazione client side.

Inoltre, per quanto questa e' una soluzione che personalmente *come utente* apprezzerei, e' un tipo di controllo a mio avviso troppo complesso (pensando alla popolazione CRI, c'e' gente con preparazione informatica quasi nulla, nonche' anziani e utenti con difficolta' cognitive), quindi l'inconsistenza con il resto dell'interfaccia utente di Gaia diventa un problema.

Suggerirei di inserire in questa pagina un semplice link (in rosso?) "Fai il reset della password per mario.rossi@regione.cri.it" che porta a una semplicissima paginetta di conferma, "Sei sicuro di voler richiedere il reset della password?" con spazio per uno o due paragrafi che spiegano le conseguenze dell'azione "Se richiederai il reset della password, ti forniremo una nuova password temporanea [...]. Una volta che avrai effettuato l'accesso con la password temporanea, ti verra' richiesto di inserire una nuova password di tua scelta [...]". Magari con due pulsanti ("Annulla" e "Continua").